doc-content/enterprise-only/processes/creating-new-process-instance-list-proc.adoc

```diff
@@ -1,5 +1,5 @@
 [id='creating-creating-new-process-instance-list-proc-{context}']
-= Creating a custom process instance list
+= Creating custom process instance filters
```

Comment:
This one would be related to advanced filters, so I would suggest something like: _Filtering process instances using advanced filters_ same applies for _4.2. Creating custom task filters_

Reply:
Ack